src/lib/projects/ApifySDKAnalyzer.ts

```diff
@@ -0,0 +1,67 @@
+import { existsSync, readFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+import { lt } from 'semver';
+
+import { CRAWLEE_PACKAGES } from './shared.js';
+
+const VERSION_WHEN_APIFY_MOVED_TO_CRAWLEE = '3.0.0';
+
+export class ApifySDKAnalyzer {
```

Comment:
so this detects SDK with versions lower than v3, right?

Reply:
Might need to document it better, but kinda? This detects
- SDK <3 for JS
- SDK presence in python _without_ crawlee also being present (so most of our python templates like scrapy, beautifulsoup, etc)